navigation/ca_move_base/launch/navigate.launch

```diff
@@ -49,4 +49,14 @@
   <include if="$(eval arg('localization')=='octomap')"
            file="$(find ca_move_base)/launch/octomap_mapping.launch"/>
 
+  <!-- Hector SLAM + map server -->
+  <include if="$(eval arg('localization')=='hector')"
```

Comment:
Shouldn't you enable `map_server`?

Reply:
Sorry, this is a misleading comment. In fact map server is already enabled, so I'll delete it from here. What i meant was that this is the option to run Hector SLAM without geotiff, so you'll need to run the map server to save the map afterwards.